tests/test_transfer_status_snapshot_report/test_transfer_status_report_base.py

```diff
@@ -132,3 +132,198 @@ def test_cutoffs(self, cutoff, registrationStatus):
 
         finally:
             self.delete_index(index_name)
+
+    @pytest.mark.parametrize(
+        "status, outcome",
+        [
+            # ("ERROR", "Not eligible for electronic transfer"),
+            ("REGISTRATION", "Not eligible for electronic transfer"),
+            # ("REGISTRATION+ERROR", "Not eligible for electronic transfer"),
+            # ("TRANSFER_COMPAT_FALSE", "Not eligible for electronic transfer"),
+            # ("TRANSFER_COMPAT_TRUE", "In progress"),
+            # ("EHR_REQUESTS", "In progress"),
+            # ("EHR_REQUESTS+ERROR", "Technical failure"),
+            # ("EHR_RESPONSES", "In progress"),
+            # ("EHR_RESPONSES+ERROR", "Technical failure"),
+            # ("READY_TO_INTEGRATE_STATUSES", "Awaiting integration"),
+            # ("EHR_INTEGRATIONS+INTEGRATION_OUTCOME_FAIL", "Technical failure"),
+            # ("EHR_INTEGRATIONS+INTEGRATION_OUTCOME_REJECT", "Rejected"),
+            # ("EHR_INTEGRATIONS+INTEGRATION_OUTCOME_INTEGRATED", "Successful integration"),
```

Comment:
Do we want to leave these commented out?

Reply:
no I will uncomment them. Thank you :)